service/src/main/java/bio/terra/policy/service/region/RegionService.java

```diff
@@ -61,11 +72,48 @@ public Region getRegion(String name) {
     return regionNameMap.get(name);
   }
 
+  public HashSet<String> getPaoDatacenters(Pao pao, String platform) {
+    List<String> regionNames = extractPolicyRegions(pao);
+    HashSet<String> result = new HashSet<>();
+
+    if (regionNames.isEmpty()) {
+      result.add(ANY_DATACENTER_INDICATOR);
```

Comment:
I think we should return the full list. The UI will use it to populate a drop-down.
Is that straightforward to get from the data structures in place?

Reply:
Sure, we can retrieve the global list.